src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -750,6 +750,23 @@ private void InitializeCustomNodeManager()
                 if (customNodeSearchRegistry.Contains(info.FunctionId))
                     return;
 
+                var elements = SearchModel.SearchEntries.OfType<CustomNodeSearchElement>().Where(
+                                x =>
+                                {
+                                    return string.Compare(x.Path, info.Path, StringComparison.OrdinalIgnoreCase) == 0;
+                                }).ToList();
+
+                if (elements.Any())
+                {
+                    foreach (var element in elements)
+                    {
+                        customNodeSearchRegistry.Remove(element.ID);
+                        element.SyncWithCustomNodeInfo(info);
+                        SearchModel.Update(element);
+                    }
+                    return;
+                }
```

Comment:
I know @aosyatnik suggested adding a `return` statement here since we're just updating it, but that seems to exclude the following `customNodeSearchRegistry.Add`, so we'll do only `customNodeSearchRegistry.Remove` but not adding it back. Am I reading this right?